src/lightning/pytorch/trainer/connectors/callback_connector.py

```diff
@@ -226,10 +226,9 @@ def _configure_external_callbacks() -> List[Callback]:
     if _PYTHON_GREATER_EQUAL_3_8_0:
         from importlib.metadata import entry_points
 
-        if _PYTHON_GREATER_EQUAL_3_10_0:
-            factories = entry_points(group=group)
-        else:
-            factories = entry_points().get(group, {})  # type: ignore[arg-type]
```

Comment:
The ignore is meant to be explicitly on this line

Reply:
Well with simplification... and this is about mypy skip so does not change much on the coding side, right?

Reply:
Yes, my comment applies specifically to mypy. You'll notice that the mypy job is failing because of this change.

Reply:
ok, I'll revisit it

Reply:
Tried to add `noqa0, but `yesqa` removed it so can we live with this one?